OntologyMapper/src/README.md

```diff
@@ -0,0 +1,253 @@
+# Microsoft.SmartPlaces.Facilities.OntologyMapper
+
+This library provides helpers useful for mapping one Digital Twins Definition Language (DTDL) model to a different DTDL model. It defines a set of classes which allow the following types to be mapped:
+
+1. Interfaces
+1. Relationships
+1. Property Projections
+1. Backfilling of Properties
+
+A json file can be created with collections of these types of mappings, and then that can be used in a process which uses one DTDL ontology as an input and outputs a different DTDL ontology.
+
+
+## File Sections
+
+### Header
+
+The header section is for informational purposes
```

Comment:
The header section is for informational purposes designating the source and destination DTDLs. This is to provide an at-a-glance summary of what DTDLs this file maps.